variables.tf

```diff
@@ -197,8 +197,16 @@ variable "reloader_custom_values" {
 variable "reloader_image" {
   type        = string
   description = "The reloader image in the format of `[registry-url]/[namespace]/[image]`."
+  default     = "stakater/reloader"
+  nullable    = false
```

Comment:
hmm, this could be set to null if `reloader_deployed` if false. Perhaps use cross variable validation?

Reply:
this seems to be a repeating problem in this module, and code is already in main branch, so maybe create a follow up issue to address in a separate PR